transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -56,7 +56,7 @@
     private long prevDeadlineNanos = NONE;
     private boolean pendingWakeup;
 
-    IOUringEventLoop(IOUringEventLoopGroup parent, Executor executor, int ringSize,
```

Comment:
How about adding little bit of Documentation or Reference so user can understand what exactly it is. Maybe 'man page' link?

Reply:
done in `IOUringEventLoopGroup`